### ISSUE TITLE
Sample matchlist for Joy Orbison - Hyph Mngo 

### ISSUE THREAD

**Issue by edgarfinn** [edgarfinn/Tune-matching-logic#2]
# Matches for Hyph Mngo by Joy Orbison (Bbm | 138bpm)

### Same pitch range (132-144bpm):
**Bbm**

Africa HiTech | Blen | Bbm | 142
MJ Cole | Sincere | Bbm | 133
Throwing Snow | Sanctum | Bbm | 132
Untold | Sweat | Bbm | 143

**Fm**

Midland | Final Credits | Fm | 133
Sticky | Things We Do For Love | Fm | 135

**Ebm** 

Bicep (feat. Ejeca) | You  | Ebm | 132
Dark star | Aidy’s Girl Is A Computer | Ebm | 135
DJ Luck & MC Neat | Little Bit Of Luck | Ebm | 135
Agent X | Decoy | Ebm | 132
Makoto | What You Want | Ebm | 135
Rusko | Love Is Real |Ebm | 140
Yasmin (feat Shy Fx & Ms Dynamite) | Light Up (The World) (MJ Cole Dub) | Ebm 133

**C#**
Jose James | Blackmagic (Joy Orbison Recreation) | C# | 134
Submerse + Resketch | Never Again | C# | 136

### Slower Pitch Range (126 - 132)

**Am**
AZ&TOR | Sit Back (Polkadot Remix) | Am | 129
Breach | Fatherless (Doc Daneeka Mrr Snrzz remix) | Am | 130
Breach | Fatherless (VIP) | Am | 128
Breach | Lets Get Hot | Am | 128
Breach | Lets Get Hot (Dub) | Am | 128
Breach & Darksky | Fallout | Am | 126
C.R.S.T | Turn For Me | Am | 130
Dantiez Saunderson & White Chocolate | Logarythm (House Of Virus Remix) | Am | 126
Die & Mensah | Firing Line | Am | 130
Foamo | There For Me | Am | 130
Fusion Groove Orchestra, Steve Lucas | If Only I Could feat. Steve Lucas (Liem Remix) Am 126
Joe Goddard | Gabriel (Gabriel Seiji Remix feat Valentina) | Am | 128
Marek Hemmann feat Fabian Reichelt | Left | Am | 127
Hot Chip | Night And Day (Dusky Remix) | Am | 126
J.D. Hall | Into You Long (Jonathan Morning Mix) | Am | 130
Jamie XX | Far Nearer | Am | 127
Leroy Peppers | Northern Nookie | Am | 128
Lynn Lockamy, Soulful Session | Hostile Takeover | Am | 126
Maddslinky | Special (feat Omar - Toddla T remix) | Am | 132
Midland | Bring Joy | Am | 130
MJ Cole | Crazy Love (Radio Edit) | Am | 132
Scruffizer | Kick It (Marshall F Remix) | Am | 127
Throwing Snow | Pyre feat Augustus Ghost | Am | 131
Unknown Artist | Anxiety (BrEaCh Remix) | Am | 128
XXXY | Ordinary Things | Am | 130

**Em**
Artful Dodger | Re rewind | Em | 130
Benny Blanco | All The Way Down (Dobies Bedroom Mute Mix) | Em | 126
Bonobo feat Andrea Triana | Eyes Down (Floating Points Remix) | Em | 130
Benjamin Damage, Doc Daneeka | Creeper (Album Edit) | Em | 128
Darksky | The Lick | Em | 132
Dimitri From Paris | Que Tal America | Em | 129
Joe Goddard | Gabriel (Gabriel Compound One remix feat Valentina)  |  Em | 127
Sidney Samson | Riverside (Afrojack remix) | Em | 127
Soulsearcher | Cant Get Enough | Em | 127
Steve Simpson | Chicagocid | Em | 126
Tru Tones | Dancing | Em | 129

**Dm**
2 Bad Mice | Limit Of Paradise | Dm | 132
Chicago | Street Player | Dm | 126
CRST | Revisit | Dm | 129
Daebull | Triatlon-Blood_Music | Dm | 131
Disclosure VS Audiojack | What's In These Days (Benny Kane Rewix) | Dm | 126
Mak & Pasteman | Fifty Shades | Dm | 126
Shadow Child | String Thing | Dm | 126
Squarehead &  Mella Dee | Get_Together-Wolf_Music_Recordings | Dm | 126
SWITCH/ANDREA MARTIN | I Still Love You (original mix) | Dm | 128
SWITCH/ANDREA MARTIN | I Still Love You (acapella) | Dm | 128

**C**
Disclosure | Tenderly | C | 128
Major Lazor | Pon De Floor | C | 127
Seiji | Face Up | C | 132

**Comment by jamesMcMeex** [comment]
So, just for the sake of clarification, have these tracks have been analysed by Serato's built-in analysis tool to dervice the root key value? 

I know that Serato analyses the BPM and saves the value to the BPM field in the ID3 tag, but does it also do the root key? My apologies, I am dusting off my Serato knowledge as it's been a while - perhaps there have been some updates.

**Comment by edgarfinn** [comment]
@jamesMcMeex Yeah thats right, in Serato you can choose how you want the key information to be displayed: either 1A, 1B, 2A etc.... ("Camelot Keys"), or "Classic keys" (ie Am, C, Fm, Ebm etc....). I've personally always favoured using classic keys, but I guess we might want to be able to offer user the choice.

<img width="638" alt="Screen Shot 2019-03-13 at 08 13 46" src="https://user-images.githubusercontent.com/15878887/54263531-f02e6b00-4568-11e9-9b0a-94421970068b.png">

**Comment by edgarfinn** [comment]
@jamesMcMeex I made this list just to get an idea for how what a Current Tune's match list could look like. I was surprised by how many results one tune gets (The above list isnt even exhaustive yet). 

Hopefully gives us a bit of food for thought on a) what the most useful info the user needs to be presented with and b) some ways of presenting that info intuitively to the user
